remix.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   // so we default back to the standard build output.
   server: process.env.NODE_ENV === "development" ? undefined : "./server.js",
   ignoredRouteFiles: ["**/.*"],
+  serverDependenciesToBundle: [/.*/],
```

Comment:
Need to add new dependencies to this array because they only support ESM. Also, had to add all dependencies because vercel fails otherwise. One of remix maintainers said they do this so its good enough for me.